src/lib/guesstimator/samplers/Simulator.js

```diff
@@ -83,12 +86,20 @@ function modularSlice(array, from, to) {
   return [...array.slice(newFrom), array.slice(0,to)]
 }
 
-function buildData(expr, prevModularIndex, numSamples, inputs) {
```

Comment:
realize this existed before, but the name ``buildData`` should be more specific.